.eslintrc.js

```diff
@@ -37,5 +37,12 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
   },
-  ignorePatterns: ['gatsby-*.js', 'jest', '*.md', 'tooling/**', 'gatsby/**'],
+  ignorePatterns: [
+    'gatsby-*.js',
+    'jest',
+    '*.md',
+    'tooling/**',
+    'gatsby/**',
+    'plugins/**',
```

Comment:
I got some errors with import statements for our js files. Can't we enable modern node to use imports? It's not optimal to remove all those files from linting.